polygon-digital-carbon/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "polygon-carbon",
-  "version": "1.2.1",
+  "version": "1.3.1",
```

Comment:
This will need to be adjusted before we merge the main PR into staging, but let's do that once we merge this in to that other PR and do final testing